featuretools/__init__.py

```diff
@@ -16,3 +16,20 @@
 from .feature_base import AggregationFeature, DirectFeature, Feature, FeatureBase, IdentityFeature, TransformFeature
 
 __version__ = '0.6.1'
+
+import pkg_resources
+import sys
+for entry_point in pkg_resources.iter_entry_points('featuretools_initialize'):
+    try:
+        module = entry_point.load()
+        if hasattr(module, 'initialize'):
+            module.initialize()
+    except Exception:
+        pass
+
+
+for entry_point in pkg_resources.iter_entry_points('featuretools_plugin'):
```

Comment:
add a comment to this block? something like

"Load in submodules registered by other libraries into Featuretools namespace"

Reply:
Added